LICENSES/MPL-2.0.txt

```diff
@@ -35,7 +35,7 @@ Mozilla Public License Version 2.0
     means any form of the work other than Source Code Form.
 
 1.7. "Larger Work"
-    means a work that combines Covered Software with other material, in 
+    means a work that combines Covered Software with other material, in
```

Comment:
I feel a bit uneasy modifying the license file, as it was downloaded by the `reuse` tool. It might check if it's the correct/unmodified file?

Could we exclude the LICENSES directory?